docker/Dockerfile

```diff
@@ -118,5 +118,5 @@ RUN \
     build-locale-archive
 
 WORKDIR /
-ENTRYPOINT ["/usr/bin/contextBroker","-fg"]
-EXPOSE 1026
\ No newline at end of file
+ENTRYPOINT ["/usr/bin/contextBroker","-fg", "-multiservice"]
```

Comment:
Maybe the fact that Orions runs in multiserve configuration should be mentioned in the Docker README.md? As a new bullet in the "A few points to consider" list.